osu.Game/Database/RealmAccess.cs

```diff
@@ -1245,6 +1246,15 @@ void remapKeyBinding(int oldAction, int newAction)
 
                     break;
                 }
+
+                case 48:
+                    const int qualified = (int)BeatmapOnlineStatus.Qualified;
+
+                    var beatmaps = migration.NewRealm.All<BeatmapInfo>().Where(b => b.StatusInt == qualified);
+
+                    foreach (var beatmap in beatmaps)
+                        beatmap.ResetOnlineInfo(resetOnlineId: false);
```

Comment:
In testing this does appear to work perhaps too well because I expected this to delete the "qualified" badge from the set in song select if there is no saved user credentials (which will fail the online lookup in `BackgroundDataStoreProcessor`). But the badge doesn't go away because the badge uses the status of the *set* rather than the individual difficulties.

A new couple of weird idiosyncrasies to add to the pile I guess...